### PR TITLE
[WIP] Review

### PR DESCRIPTION
**Pull request by kbaley**
Not much to do here. It's pretty solid.

### CODE REVIEW

File: _posts/2019-03-26-read-models-a-knot-to-untie.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ class SampleHandler : IHandleRequests
 }
 ```
 
-the infrastructure executing the above code could, at runtime, store the resulting ViewModel, the `dynamic vm`, in a dedicated incredibly fast storage, and query that storage for all the subsequent requests for the same URL. We've basically created a read model at runtime at the first request. So far, so good.
+the infrastructure executing the above code could, at runtime, store the resulting ViewModel, the `dynamic vm`, in a dedicated incredibly fast storage, and query that storage for all subsequent requests for the same URL. We've basically created a read model at runtime at the first request. So far, so good.
 
 ### You know what?
```
Comment (kbaley) [inline _posts/2019-03-26-read-models-a-knot-to-untie.md]: Maybe consider a more descriptive title than "Ownership dilemma". It's not immediately obvious in the transition that you're about to talk about the problems with caching.

File: _posts/2019-03-26-read-models-a-knot-to-untie.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ category: view-model-composition
 redirect_from: "/2019/03/26/read-models-a-knot-to-untie.html"
 ---
 
-When dealing with distributed systems it's very tempting, and apparently easy and simple, to solve the "we need to present data to users" problem building read models. Read models designed to satisfy presentation requirements, containing data coming from different services.
+When dealing with distributed systems it's very tempting, and apparently easy and simple, to solve the "we need to present data to users" problem by building read models. Read models are designed to satisfy presentation requirements, containing data coming from different services.
```
Comment (mauroservienti) [inline _posts/2019-03-26-read-models-a-knot-to-untie.md]: @kbaley As I read it, it sounds like it has a different meaning compared to my original intentions.
The second sentence is referred to the previous one, it specifies that the mentioned read models are designed to compose data coming from different services. The “are” turns it into a general statement, and it should not be.

File: _posts/2019-03-26-read-models-a-knot-to-untie.md (hunk 1 above)
Comment (mauroservienti) [reply _posts/2019-03-26-read-models-a-knot-to-untie.md]: Changed

**Comment by mauroservienti** [comment]
Cherry picked all suggested changes. There is still a pending comment.

**Pull request closed without merging**